cmd/cmd.go

```diff
@@ -797,6 +799,39 @@ func RunServer(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	if d := os.Getenv("OLLAMA_DISCOVERY"); d == "ENABLED" {
+		if host != "0.0.0.0" {
+			host = "0.0.0.0"
```

Comment:
For Network Discovery to work the service needs to be running on the `0.0.0.0` instead of just the `127.0.0.1` loopback.